GBz80.xml

```diff
@@ -1,19 +1,19 @@
 <NotepadPlus>
-    <UserLang name="GBz80" ext="asm inc" udlVersion="2.1">
+    <UserLang name="GBz80" ext="asm inc z80" udlVersion="2.1">

```

Comment:
This is likely to mistakenly color actual Z80 files as RGBASM, so I'm not fond of it.

Additionally, I agree that the language name should be changed:
```suggestion
    <UserLang name="RGBASM" ext="asm inc" udlVersion="2.1">
```

Reply:
Yep! That makes sense.